generator/syndesis-template.go

```diff
@@ -141,10 +144,10 @@ var productContext = Context{
 	},
 	Tags: tags{
 		Postgresql:    "9.5",
-		PemToKeystore: "1.0",
-		Keycloak:      "1.0",
-		Syndesis:      "1.0",
-		Atlasmap:      "1.30",
+		PemToKeystore: "1.0.0",
+		Syndesis:      "1.0.0",
+		Atlasmap:      "1.30.0",
+		OAuthProxy:    "v1.0.0",
```

Comment:
Is the OAuthProxy supposed to be part of our Monorepo ? If so, I would like to align the version numbering, using a '1.0.0' (or as laid out above a 1.0)

Reply:
No, the oauth proxy is meant to be an external image direct provided by OpenShift itself. We should not be hacking on it as part of Syndesis.

Reply:
ok, got it.